### PR TITLE
Fixed deprecation message for seo twig extension

### PR DESCRIPTION
**Pull request by alexander-schranz**
| Q | A
| --- | ---
| Bug fix? | yes
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| License | MIT

#### What's in this PR?

Fixed deprecation message for seo twig extension.

#### Why?

Currently an error could happen based on the php settings.

### CODE REVIEW

**Comment by danrot** [comment]
@alexander-schranz Don't know what happened here, but the build was failing :see_no_evil:

**Comment by alexander-schranz** [comment]
@danrot did run the tests on the current master branch which was green 8 days ago but know fails with the same errors: https://travis-ci.org/sulu/sulu/jobs/429891387.

Maybe some changes in the symfony 4 compatibility had unknown bc breaks /cc @wachterjohannes

**Comment by alexander-schranz** [comment]
I added jackalope 1.3.6 currently as conflict because of the build is failing on jackrabbit there because of this changes: https://github.com/jackalope/jackalope/pull/351

File: src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php [hunk 1]
```diff
@@ -80,15 +80,15 @@ public function renderSeoTags(
...
             'This twig extension is deprecated and should not be used anymore, include the "%s".',
             $template
-        );
+        ));
 
         $defaultLocale = null;
         $portal = $this->requestAnalyzer->getPortal();
         if ($portal) {
-            $defaultLocale = $portal->getXDefaultLocalization()->getLocale();
+            $defaultLocale = $portal->getDefaultLocalization()->getLocale();
```
Comment (danrot) [inline src/Sulu/Bundle/WebsiteBundle/Twig/Seo/SeoTwigExtension.php]: Why should we use the other default localization here? I thought the `x-default` was introduced especially for the SEO functionality.

File: composer.json [hunk 2]
```diff
@@ -68,7 +68,7 @@
         "symfony/symfony": "2.8.10 || 3.4.12",
         "symfony/phpunit-bridge": "2.8.10",
         "symfony/monolog-bundle": "2.8.10",
-        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1",
+        "jackalope/jackalope": "1.2.8 || 1.3.0 || 1.3.1 || 1.3.6",
```
Comment (danrot) [inline composer.json]: Do they know and do something about this issue?

Comment (alexander-schranz) [reply composer.json]: Didn't got any feedback yet from @dbu or @middlebrain

Comment (alexander-schranz) [reply composer.json]: created also an issue in jackrabbit repository as there the build fails with the same error: https://github.com/jackalope/jackalope-jackrabbit/issues/156

**Pull request merged**